lighthouse-core/audits/dobetterweb/has-inspector-issues.js

```diff
@@ -0,0 +1,192 @@
+/**
+ * @license Copyright 2020 The Lighthouse Authors. All Rights Reserved.
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with the License. You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the specific language governing permissions and limitations under the License.
+ */
+
+/**
+ * @fileoverview Audits a page to determine whether it generates issues in the Issues panel of Chrome Devtools.
+ * The audit is meant to maintain parity with the Chrome Devtools Issues panel front end.
+ * https://source.chromium.org/chromium/chromium/src/+/master:third_party/devtools-frontend/src/front_end/sdk/
+ */
+
+'use strict';
+
+/** @typedef {{url: string}} IssueSubItem */
+/** @typedef {{issueType: string|LH.IcuMessage, subItems: Array<IssueSubItem>}} IssueItem */
+
+const Audit = require('../audit.js');
+const i18n = require('../../lib/i18n/i18n.js');
+
+const UIStrings = {
+  /** Title of a Lighthouse audit that provides detail on various types of problems with a website, like security or network errors. This descriptive title is shown to users when no issues were logged into the Chrome DevTools Issues panel. */
+  title: 'No issues in the `Issues` panel in Chrome Devtools',
+  /** Title of a Lighthouse audit that provides detail on various types of problems with a website, like security or network errors. This descriptive title is shown to users when issues are detected and logged into the Chrome DevTools Issues panel. */
+  failureTitle: 'Issues were logged in the `Issues` panel in Chrome Devtools',
+  /* eslint-disable max-len */
+  /** Description of a Lighthouse audit that tells the user why issues being logged to the Chrome DevTools Issues panel are a cause for concern and so should be fixed. This is displayed after a user expands the section to see more. No character length limits. */
+  description: 'Issues logged to the `Issues` panel in Chrome Devtools indicate unresolved problems. They can come from network request failures, insufficient security controls, and other browser concerns. Open up the Issues panel in Chrome DevTools for more details on each issue.',
+  /* eslint-enable max-len */
+  /** Table column header for the types of problems observed in a website, like security or network errors. */
+  columnIssueType: 'Issue type',
+  /** The type of an Issue in Chrome DevTools when a resource is blocked due to receiving a rejection in the response to a request made on the page. */
+  issueTypeBlockedByResponse: 'Blocked by response',
+  /** The type of an Issue in Chrome DevTools when a site has large ads that use up a lot of the browser's resources. */
+  issueTypeHeavyAds: 'Heavy resource usage by ads',
```

Comment:
Is this a better alternative? @brendankenny

Reply:
Also, rethinking this string has made me rethink the blockByResponse string as well. When I think about it, I don't know how much 'blocked by response' makes sense here since it's really the response that is being blocked if I understand the issue correctly. Thoughts?

Reply:
> Is this a better alternative? @brendankenny

yes, that's great, I think.



> Also, rethinking this string has made me rethink the blockByResponse string as well. When I think about it, I don't know how much 'blocked by response' makes sense here since it's really the response that is being blocked if I understand the issue correctly. Thoughts?

Yeah, now that you mention it, I feel the same way. It definitely sounds confusing and not like something that should be possible. I don't have a great suggestion for an alternative, though :/